logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageContent.kt

```diff
@@ -1,10 +1,17 @@
 package com.wire.kalium.logic.data.message
 
+import com.wire.kalium.protobuf.messages.QualifiedConversationId
+
 sealed class MessageContent {
     data class Text(val value: String) : MessageContent()
     data class Calling(val value: String) : MessageContent()
     data class Asset(val value: AssetContent) : MessageContent()
     data class DeleteMessage(val messageId: String) : MessageContent()
-    data class DeleteForMe(val messageId: String) : MessageContent()
+    data class DeleteForMe(
+        val messageId: String,
+        val conversationId: String,
+        val qualifiedConversationId: QualifiedConversationId? = null
```

Comment:
Perhaps we should not make it optional.

We support federation, and we need to have this support everywhere.
If we want to make it nullable, we better explicitly pass `null`. But just making it implicit is too easy to forget.